modules/tinymce/src/models/dom/main/ts/table/selection/Ephemera.ts

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (c) Tiny Technologies, Inc. All rights reserved.
+ * Licensed under the LGPL or a commercial license.
+ * For LGPL see License.txt in the project root for license information.
+ * For commercial licenses see https://www.tiny.cloud/
+ */
+
+/*
+ NOTE: This file is duplicated in the following locations:
+  - core/table/TableEphemera.ts
```

Comment:
It doesn't look like this exists anymore, so this probably should be removed? The other files also need updating.

Reply:
Have fixed accidentally leave the core references in for the Ephemera, Utils and TableSelection files